src/scripting_api/initialization.js

```diff
@@ -21,7 +21,7 @@ import { Field } from "./field.js";
 import { ProxyHandler } from "./proxy.js";
 import { Util } from "./util.js";
 
-function initSandbox(data, extra, out) {
+function initSandbox(data, extra, out, testMode = false) {
```

Comment:
Given the increasing number of parameters here, it might not be a bad idea to change this to take a parameter object instead!?